ietf/meeting/management/commands/import_iesg_minutes.py

```diff
@@ -0,0 +1,301 @@
+# Copyright The IETF Trust 2023, All Rights Reserved
+
+from collections import namedtuple
+import datetime
+import os
+import re
+import shutil
+
+from django.conf import settings
+from django.core.management import BaseCommand
+
+from pathlib import Path
+from zoneinfo import ZoneInfo
+from ietf.doc.models import DocEvent, Document
+
+from ietf.meeting.models import (
+    Meeting,
+    SchedTimeSessAssignment,
+    Schedule,
+    SchedulingEvent,
+    Session,
+    TimeSlot,
+)
+from ietf.name.models import DocTypeName
+
+
+def add_time_of_day(bare_datetime):
+    """Add a time for the iesg meeting based on a date and make it tzaware
+
+    From the secretariat - the telechats happened at these times:
+    2015-04-09 to present: 0700 PT America/Los Angeles
+    1993-02-01 to 2015-03-12: 1130 ET America/New York
+    1991-07-30 to 1993-01-25: 1200 ET America/New York
+    """
+    dt = None
+    if bare_datetime.year > 2015:
+        dt = bare_datetime.replace(hour=7).astimezone(ZoneInfo("America/Los_Angeles"))
```

Comment:
As I describe in my long comment below, you want to use `replace()` instead of `astimezone()` to set the tzinfo on a naive datetime.